cmd/snap/cmd_debug_state.go

```diff
@@ -78,11 +80,12 @@ func init() {
 		return &cmdDebugState{}
 	}, timeDescs.also(map[string]string{
 		// TRANSLATORS: This should not start with a lowercase letter.
-		"change":  i18n.G("ID of the change to inspect"),
-		"task":    i18n.G("ID of the task to inspect"),
-		"dot":     i18n.G("Dot (graphviz) output"),
-		"no-hold": i18n.G("Omit tasks in 'Hold' state in the change output"),
-		"changes": i18n.G("List all changes"),
+		"change":    i18n.G("ID of the change to inspect"),
+		"task":      i18n.G("ID of the task to inspect"),
+		"dot":       i18n.G("Dot (graphviz) output"),
+		"no-hold":   i18n.G("Omit tasks in 'Hold' state in the change output"),
+		"changes":   i18n.G("List all changes"),
+		"is-seeded": i18n.G("Output is seeded successfully"),
```

Comment:
Output? I would expect something like `System is seeded ...`

Reply:
I updated the text now.

Reply:
Can you check, it looks like the same text is used.

Reply:
Can you please reload?